src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/ProcessFullyApprovedCohort/ProcessFullyApprovedCohortCommandHandler.cs

```diff
@@ -46,8 +46,8 @@ protected override async Task Handle(ProcessFullyApprovedCohortCommand request,
                 .Select(a => new ApprenticeshipCreatedEvent
                 {
                     ApprenticeshipId = a.Id,
-                    CreatedOn = a.Cohort.TransferApprovalActionedOn ?? a.AgreedOn.Value,
-                    AgreedOn = a.AgreedOn.Value,
+                    CreatedOn = a.Cohort.LastUpdatedOn,
+                    AgreedOn = a.Cohort.LastUpdatedOn,
                     AccountId = a.Cohort.EmployerAccountId,
```

Comment:
Is LastUpdatedOn updated when the transfer is approved. And as you mentioned earlier we may have to differentiate between when the 2 parties agreed versus when the transferSender finally approved